src/layout.scss

```diff
@@ -1,5 +1,5 @@
-@import "../scss/layout/container";
-@import "../scss/layout/col";
-@import "../scss/layout/ui";
-@import "../scss/layout/shell";
-@import "../scss/layout/app";
+@import "./layout/container";
+@import "./layout/col";
+@import "./layout/ui";
+@import "./layout/shell";
+@import "./layout/app";
```

Comment:
should these layout ones be grouped into a scss/css that can deliver them?

Reply:
That is the purpose of this file (if I understand the question)